.github/workflows/trino-integration-test.yml

```diff
@@ -76,7 +76,7 @@ jobs:
 
       - name: Package Gravitino
         run: |
-          ./gradlew compileDistribution -x test -PjdkVersion=${{ matrix.java-version }}
+          ./gradlew compileDistribution compileTrinoConnector -x test -PjdkVersion=${{ matrix.java-version }}
```

Comment:
Is the task `compileTrinoConnector` NOT included in `compileDistribution`?

Reply:
Yes, the `compileTrinoConnector`  task does not include `compileDistribution`